docs/user/cli/console.rst

```diff
@@ -0,0 +1,99 @@
+GeoMesa Scala Console
+=====================
+
+The GeoMesa tools have a ``scala-console`` command that starts a Scala REPL configured
+for use with GeoMesa. The command will place the GeoMesa classpath and configuration
+for that distribution on the REPL's classpath. Additionally it will preload commonly
+used imports. The command also comes with tooling that will provide the option to
+download and run the appropriate version Scala if it is not available. This is a
+portable install and will not alter the machines current configuration.
+
+Example usage
+-------------
+
+The following example was performed against the FileSystem Datastore generated by the
+:ref:`fsds_quickstart`. It shows how to use the ``scala-console`` command to connect
+to a FileSystem Datastore, discover the feature type names, get the schema and query for data.
+
+.. tabs::
+
+    .. tab:: Demo
+
+        .. raw:: html
+
+            <script src="https://asciinema.org/a/162999.js" id="asciicast-162999"  async></script>
+
+    .. tab:: Transcript
+
+        First we run the command, this starts up the Scala REPL with all of the
+        resources we need.
+
+        .. code-block:: scala
+
+            aheyne@aheyne:/tmp/geomesa-fs_2.11-2.0.0-SNAPSHOT$ bin/geomesa-fs scala-console
```

Comment:
could you change the `aheyne@aheyne:` to just be `$`? that's what we do other places. Also, do you think it makes sense to pull the command up into a distinct bash code block, then use scala for the output?

Reply:
Yeah, I updated the asciinema to only be `$` as well.